runtime/tutorials/tutor.tutor

```diff
@@ -207,7 +207,7 @@ is displayed:
 
 Commands to run in the system shell can be highlighted by indenting a line starting with "$".
 
-~~~ sh
+~~~
```

Comment:
This is because `vim --version` isn't a shell script, so shouldn't be highlighted as such.